geometry_msgs/Point map_point
---
geographic_msgs/GeoPoint ll_point
